src/util/gobConfig.ml

```diff
@@ -84,6 +84,14 @@ sig
 
   (** Add a schema to the conf*)
   val addenum_sch: jvalue -> unit
+
+  (** Whether for a given ikind, we should compute with wrap-around arithmetic.
+    *  Always for unsigned types, for signed types if 'sem.int.signed_overflow' is 'assume_wraparound'  *)
+  val should_wrap: Cil.ikind -> bool
+
+  (** Whether for a given ikind, we should assume there are no overflows.
+    * Always false for unsigned types, true for signed types if 'sem.int.signed_overflow' is 'assume_none'  *)
+  val should_ignore_overflow: Cil.ikind -> bool
```

Comment:
I think these C and analysis specific things shouldn't be in `GobConfig`, which otherwise implements an independent config system. These should be in `IntDomain` or some `IntUtil`.